core/optionsx.h

```diff
@@ -1627,8 +1627,11 @@ OPTION_DEFAULT(uint_size, vmheap_size_wow64, 128 * 1024 * 1024,
                "capacity of virtual memory region reserved for unreachable heap "
                "on WoW64 processes")
 #endif
-/* We hardcode an address in the mmap_text region here, but verify via
- * in vmk_init().
+/* We hardcode a default vmcode base address, to which we add a random offset
+ * up to vm_max_offset.
+ * If -vm_base is set to 0, and -vm_base_near_app is off, we let the OS pick
```

Comment:
Shouldn't this be in the user-visible option comment?

Reply:
This comment is all there is.  There is no Doxygen for these.  A handful of options are in the docs in `sec_options` but not this one since we don't expect most users to set this.